test/mock/rtps/BuiltinProtocols/fastdds/rtps/builtin/BuiltinProtocols.h

```diff
@@ -76,8 +77,8 @@ class BuiltinProtocols
     //!Pointer to the WLP
     WLP* mp_WLP;
 
-    //!Pointer to the TypeLookupManager
-    fastdds::dds::builtin::TypeLookupManager* tlm_;
+    //! TypeLookupManager
+    fastdds::dds::builtin::TypeLookupManager typelookup_manager_;
```

Comment:
Why this change is here? It's already on `feature/xtypes1.3` branch?

Reply:
Yes, this is something already merged in Fast DDS XTypes development branch but it is going to be reverted. But until then, in order to build the tests using this mock, the change is required.